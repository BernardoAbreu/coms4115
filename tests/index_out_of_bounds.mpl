a = [1, 2, 3, 4, 5];

b = a[7];

print(b);