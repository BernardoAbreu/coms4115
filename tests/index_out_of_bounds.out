Array out of bounds exception
